src/components/SignupForm.vue

```diff
@@ -0,0 +1,412 @@
+<script setup>
+  import axios from 'axios'
+  import 'vue3-toastify/dist/index.css';
+  import { useRouter } from 'vue-router';
+  import { useToastStore } from '@/stores/toast.store';
+
+  const router = useRouter();
+  const toastStore = useToastStore();
+
+  let pseudoSignup = ""
+  let emailSignup = ""
+  let passwordSignup = ""
+
+  function signup() {    
```

Comment:
Use the folder api and create an index of call for a specific route, do not make call directly on the component. WIP for later.

Reply:
Skip it for now tho.